cmd/goal/account.go

```diff
@@ -471,11 +477,36 @@ var changeOnlineCmd = &cobra.Command{
 	Long:  `Change online status for the specified account. Set online should be 1 to set online, 0 to set offline. The broadcast transaction will be valid for a limited number of rounds. goal will provide the TXID of the transaction if successful. Going online requires that the given account have a valid participation key.`,
```

Comment:
While here, can we fix the help-text ('Going online requires that the given account have...' : have -> has)?

Also it might be confusing to people that this command does not 'install' the part file too - we should try to make sure there is no confusion.

Reply:
Indeed, clarified both.